### ISSUE TITLE
Build a view that shows the project's invoices (front-end)

### ISSUE THREAD

**Issue by dancameron** [SproutApps/sprout-invoices-project-panorama-integration#2]
Front-end and backend

**Comment by dancameron** [comment]
@3pointross I'll build the admin and the front-end views but I think I need your help. I'm not sure if PP has an API for meta boxes or not, if it does can you create one for me to use (just a blank one), then I'll finish it off. Same goes for the front end table view that I'll create, a placeholder would be helpful.

**Comment by 3pointross** [comment]
@dancameron What fields do you see going into the meta box? I'm trying to decide if it makes more sense to add the fields into the existing meta boxes via a filter or have a stand alone meta box.

The front end table placeholder is straight forward.

**Comment by dancameron** [comment]
@3pointross I think a standalone meta box would be ideal. If you don't have a wrapper API for metaboxes than I can use my own, since both plugins will need to be present; I just wanted to keep the plugin more PP centric on the dev side.

**Comment by 3pointross** [comment]
I just pushed a placeholder widget for the PSP dashboard UI, accessible at domain.com/panorama (unless the slug is changed in settings.) 

It's located in views/panorama/pspsi-dashboard-widget.php. If you can get the tables of invoices markup on the page I can handle the styling.

**Comment by dancameron** [comment]
@3pointross I created a front-end view but I'll need your help making it look good.

**Comment by 3pointross** [comment]
This is looking good, I've made some good headway on styling. Do you happen to have any dummy data that you can export? I want to make sure I'm considering how it's displayed with all use cases. 

If not I'll manage.

Best
-Ross

**Comment by dancameron** [comment]
@3pointross I actually don't have much dummy posts for the integration site I'm using. Instead I just created a couple invoices, estimates, and a client. I'm sure you've already created more than I have by now.

I'll do a review after, so you don't need to worry too much.

**Comment by 3pointross** [comment]
@dancameron Just pushed the front end work for the projects metabox and front end. I added some new hooks in Panorama to better locate the estimates / templates so [you'll want to update your version to this one](https://www.dropbox.com/s/ar58lmx3ty33prd/project-panorama.1.3.5.zip?dl=0). 

I have some ideas how to improve the front end experience but this should do for version one. Let me know if I missed the styling of any key elements -- I tried as many variations and actions as I could find.
